src/Actions/Pending.ts

```diff
@@ -134,6 +136,40 @@ const hasPendingGeth = (
   });
 };
 
+/**
+ * Uses the locally maintained TxPool to check
+ * for pending transactions in the transaction pool.
+ * @param {Config} conf Config object.
+ * @param {TransactionRequest} txRequest
+ * @param {string} type (optional) Type of pending request: claim,execute.
+ * @param {boolean} checkGasPrice (optional, default: true) Check if transaction's gasPrice is sufficient for Network.
+ * @param {number} minPrice (optional) Expected gasPrice.
+ * @returns {Promise<object>} Transaction, if a pending transaction to this address exists.
+ */
+const hasPendingPool = (
+  conf: Config,
+  txRequest: ITxRequestPending,
+  opts: PendingOpts
+): Promise<boolean> => {
+  opts.checkGasPrice = opts.checkGasPrice === undefined ? true : opts.checkGasPrice;
```

Comment:
change the checkGasPrice in PendingOps to bool from nullable bool. By default you will get false. After that change this line can be removed.